zerver/openapi/zulip.yaml

```diff
@@ -8416,6 +8416,63 @@ paths:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
 
+  /users/{user_id}/status:
+    get:
+      operationId: get-user-status
+      summary: Get a user's status
+      tags: ["users"]
+      description: |
+        Get the [status](/help/status-and-availability) of a user.
+
+        **Changes**: New in Zulip 9.0 (feature level 262).
+      parameters:
+        - $ref: "#/components/parameters/UserId"
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      status:
+                        allOf:
+                          - description: |
+                              The status of the user.
```

Comment:
I think we should note what will be returned if the user hasn't set a status here, e.g. an empty dict. If you read through all the fields, they all note that they may or may not be present. But it's probably worth noting that at the top here.

And it should be clear that the user has set their status.

Reply:
I have changed the description:
```
The status of the user. If the user has not set their status, an empty dictionary will be returned.
```

Reply:
Let's be a bit more clear. Revision:
```
The status set by the user. Note that, if the user doesn't have a status
currently set, then the returned dictionary will be empty as none of the
keys listed below will be present.
```